README.md

```diff
@@ -145,13 +146,28 @@ Facilitators:
 - Stephen Augustus ([@justaugustus](https://github.com/justaugustus)), Cisco
 - Zach Corleissen ([@zacharysarah](https://github.com/zacharysarah)), Linux Foundation
 
-### GitHub Management
+### Inclusiveness Team
 
-Status: [Proposed](https://github.com/cncf/tag-contributor-strategy/issues/5)
+Facilitators
 
-Facilitators:
+- Melania De Angelis ([@melaniadeangelis](https://github.com/melaniadeangelis))
+- Sara Trapetti ([@SarahTrap](https://github.com/SarahTrap))
+- Jade Applegate ([@jadeapplegate](https://github.com/jadeapplegate))
 
-- Gerred Dillon ([@gerred](https://github.com/gerred)), Mycelial
-- Ihor Dvoretskyi ([@idvoretskyi](https://github.com/idvoretskyi)), CNCF
+### Mentoring WG
+
+Status: [Proposed](https://github.com/cncf/tag-contributor-strategy/issues/167)
+
+Facilitators
+
+- Nate W ([@nate-double-u](https://github.com/nate-double-u)), CNCF
+- Ihor Dvoretskyi ([@idvoretskyi](https://github.com/idvoretskyi))
 - Josh Berkus ([@jberkus](https://github.com/jberkus)), Red Hat
-- Stephen Augustus ([@justaugustus](https://github.com/justaugustus)), Cisco
+
+
+### Community Infrastructure Team
+
```

Comment:
Once we have the intro paragraphs for these teams put together, we can come back and update this page with more info on what they are and how to engage with them.